akrivis-ingestor/src/main/java/org/kie/akrivis/scheduler/responses/Result.java

```diff
@@ -0,0 +1,17 @@
+package org.kie.akrivis.scheduler.responses;
+
+import org.akrivis.dbmodel.result.Threshold;
+
+public class Result {
```

Comment:
For all these classes with public fields we could use Java records can't we?

It's unsupported in Hibernate though

Reply:
This is true. Records are a cleaner solution. I will edit.